Content.Shared/_RMC14/Areas/AreaSystem.cs

```diff
@@ -339,13 +362,16 @@ public override void Update(float frameTime)
                     }
 
                     if (areaGrid.Areas.TryGetValue(pos, out var area) &&
-                        area.TryGet(out var areaComp, _prototypes, _compFactory) &&
-                        areaComp.MinimapColor != default)
+                        area.TryGet(out var areaComp, _prototypes, _compFactory))
                     {
-                        areaGrid.Colors[pos] = areaComp.MinimapColor.WithAlpha(0.5f);
-                        continue;
-                    }
+                        AddBuildableTile((ent, areaGrid), area);
 
+                        if (areaComp.MinimapColor != default)
+                        {
+                            areaGrid.Colors[pos] = areaComp.MinimapColor.WithAlpha(0.5f);
+                            continue;
+                        }
+                    }
```

Comment:
Same here

Reply:
Done in [`b467dc2` (#6288)](https://github.com/RMC-14/RMC-14/pull/6288/commits/b467dc25bba4b47429a448cfd299a99ee52238a4)